.github/README.md

```diff
@@ -60,26 +82,39 @@ After that, you can just open the `UndertaleModTool.sln` file in Visual Studio,
 You can adjust win-x86 to other RIDs, such as win-x64 or win-arm.
 
-### data.win file format
+# data.win File Format
+
 Interested in the file and instruction format research I've done while working on this? Check out these:
-* https://github.com/krzys-h/UndertaleModTool/wiki/Corrections-to-GameMaker-Studio-1.4-data.win-format-and-VM-bytecode,-.yydebug-format-and-debugger-instructions
-* https://github.com/krzys-h/UndertaleModTool/wiki/Changes-in-GameMaker-Studio-2
-* https://github.com/krzys-h/UndertaleModTool/wiki/Extensions,-Shaders,-Timelines-format
-* https://github.com/krzys-h/UndertaleModTool/wiki/Bytecode-version-differences
-* https://github.com/krzys-h/UndertaleModTool/wiki/YYC-games
 
-### Special thanks
+* [Corrections to GameMaker Studio 1.4 data.win format and VM bytecode, .yydebug format and debugger instructions](https://github.com/krzys-h/UndertaleModTool/wiki/Corrections-to-GameMaker-Studio-1.4-data.win-format-and-VM-bytecode,-.yydebug-format-and-debugger-instructions)
+* [Changes in GameMaker Studio 2](https://github.com/krzys-h/UndertaleModTool/wiki/Changes-in-GameMaker-Studio-2)
+* [Extensions, Shaders, Timelines format](https://github.com/krzys-h/UndertaleModTool/wiki/Extensions,-Shaders,-Timelines-format)
+* [Bytecode version differences](https://github.com/krzys-h/UndertaleModTool/wiki/Bytecode-version-differences)
+* [YYC games](https://github.com/krzys-h/UndertaleModTool/wiki/YYC-games)
+
+feel free to check out the [Wiki](https://github.com/krzys-h/UndertaleModTool/wiki) for more.
+
+# Special thanks
+
 Special thanks to everybody who did previous research on unpacking and decompiling Undertale, it was a really huge help:
-* https://pcy.ulyssis.be/undertale/
-* [https://github.com/donkeybonks/acolyte/wiki/Bytecode](https://web.archive.org/web/20191126144953if_/https://github.com/donkeybonks/acolyte/wiki/Bytecode)
-* https://github.com/PoroCYon/Altar.NET
-* https://github.com/WarlockD/GMdsam
+
+* [Ulyssis's UNDERTALE decompilation research](https://pcy.ulyssis.be/undertale/)
+* [Donkeybonks's GameMaker data.win Bytecode research](https://web.archive.org/web/20191126144953if_/https://github.com/donkeybonks/acolyte/wiki/Bytecode)
+* [PoroCYon's Altar.NET](https://github.com/PoroCYon/Altar.NET)
+* [WarlockD's GMdsam](https://github.com/WarlockD/GMdsam)
 * [@NarryG](https://github.com/NarryG) for [helping me figure out](https://github.com/krzys-h/UndertaleModTool/issues/3) the missing stuff for GMS2 and Nintendo Switch release
 * [@colinator27](https://github.com/colinator27) for [lots of things, including the gml compiler](https://github.com/krzys-h/UndertaleModTool/issues/4), [Sha](https://github.com/krzys-h/UndertaleModTool/issues/13)[ders](https://github.com/krzys-h/UndertaleModTool/pull/25) and [a bunch of other stuff](https://github.com/krzys-h/UndertaleModTool/pull/30)
 * [@Kneesnap](https://github.com/Kneesnap) for [improving the decompiler a bunch](https://github.com/krzys-h/UndertaleModTool/pull/162)
```

Comment:
I'd personally omit these from the special thanks, as they're already covered by the "all contributors" list down below.